Airports/EGTF/Positions.txt

```diff
@@ -0,0 +1 @@
+EGTF_I_TWR:Fairoaks Information:123.425:TFT:S:EGTF:TWR:-:-:::N051.20.53.000:W000.33.31.000::
```

Comment:
There is also a radio position on the same frequency - doesn't do any harm adding it now since it we probably won't come back to this for years so might as well do it now.
![image](https://user-images.githubusercontent.com/69194446/160299387-6cb925cc-6d9c-4949-9e14-c755468914b6.png)
```suggestion
EGTF_I_TWR:Fairoaks Information:123.425:TFT:S:EGTF:TWR:-:-:::N051.20.53.000:W000.33.31.000::
EGTF_R_TWR:Fairoaks Radio:123.425:TFT:S:EGTF:TWR:-:-:::N051.20.53.000:W000.33.31.000::
```